examples/gauge/derived-gauge-quickstart.js

```diff
@@ -54,6 +54,8 @@ const gauge = metricRegistry.addDerivedInt64Gauge('active_handles_total', metric
 const queue = new QueueManager();
 queue.addJob();
 
-// The value of the gauge is observed from the obj whenever metrics are
+// The value of the gauge is observed from a function whenever metrics are
 // collected. In this case it will be 1.
-gauge.createTimeSeries(labelValues, queue);
+gauge.createTimeSeries(labelValues, () => {
```

Comment:
Can this just be `queue.getPendingJobs` with no need for a wrapper arrow function?

Reply:
Done.